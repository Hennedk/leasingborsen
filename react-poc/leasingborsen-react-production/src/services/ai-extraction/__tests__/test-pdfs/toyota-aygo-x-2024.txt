Toyota Privatleasing
==================

Gyldig fra: 1. juni 2024
Alle priser inkl. moms og afgifter

Toyota Aygo X
=============

Løbetid: 48 måneder
Årlig kørsel: 15.000 km
Førstegangsydelse: 3 x månedsydelse

Techniske specifikationer:
- Motor: 1,0 liter VVT-i
- Effekt: 72 hk / 53 kW
- Transmission: CVT automatgear
- Brændstofforbrug: 18,5 km/l (blandet kørsel)
- CO2-udslip: 123 g/km
- 0-100 km/t: 14,5 sekunder

Variant 1: Aygo X 1.0 VVT-i X-trend CVT
- Månedsydelse: 2.899 kr
- Førstegangsydelse: 8.697 kr  
- CO2-afgift (halvårlig): 330 kr
- Energimærke: B

Variant 2: Aygo X 1.0 VVT-i X-cite CVT
- Månedsydelse: 3.199 kr
- Førstegangsydelse: 9.597 kr
- CO2-afgift (halvårlig): 330 kr
- Energimærke: B

Variant 3: Aygo X 1.0 VVT-i X-clusiv CVT
- Månedsydelse: 3.399 kr
- Førstegangsydelse: 10.197 kr
- CO2-afgift (halvårlig): 330 kr
- Energimærke: B
- Ekstra udstyr: Panoramatag, sportssæder

Tilvalg og pakker:
=================

Hjulpakker:
- Vinterhjul 15": 299 kr/måned
- Vinterhjul 16": 349 kr/måned
- Alufælge 17" sportspakke: 450 kr/måned

Service og forsikring:
- Service & Vedligeholdelse: 450 kr/måned
- Kasko forsikring: 650 kr/måned
- Ansvarsforsikring: 150 kr/måned

Øvrige tilvalg:
- Metallic lak: 125 kr/måned
- Tågelygter: 75 kr/måned
- Aircondition: 200 kr/måned

Vilkår og betingelser:
=====================

- Alle priser er inkl. 25% moms
- Priserne er vejledende og kan ændres uden varsel
- Mindste alder: 23 år
- Fast månedlig ydelse gennem hele løbetiden
- Ingen restværdi ved kontraktens udløb
- Kilometertillæg: 2,50 kr/km ved overskridelse

Kontaktoplysninger:
==================

Toyota Denmark
Telefon: 70 10 20 30
E-mail: leasing@toyota.dk
Website: www.toyota.dk/privatleasing

Dokumentdato: 1. juni 2024
Dokumenttype: Privatleasing prisliste
Valuta: DKK (Danske kroner)
Sprog: Dansk