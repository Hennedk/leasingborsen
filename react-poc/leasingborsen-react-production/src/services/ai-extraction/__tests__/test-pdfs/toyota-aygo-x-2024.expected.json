{
  "documentInfo": {
    "brand": "Toyota",
    "documentDate": "2024-06-01",
    "currency": "DKK",
    "language": "da",
    "documentType": "private_leasing"
  },
  "vehicles": [
    {
      "model": "Aygo X",
      "leasePeriodMonths": 48,
      "powertrainType": "gasoline",
      "variants": [
        {
          "variantName": "X-trend CVT",
          "engineSpecification": "1.0 VVT-i - 72 hk",
          "transmission": "cvt",
          "pricing": {
            "monthlyPayment": 2899,
            "firstPayment": 8697,
            "annualKilometers": 15000,
            "co2TaxBiannual": 330
          },
          "specifications": {
            "fuelConsumptionKmpl": 18.5,
            "co2EmissionsGkm": 123,
            "energyLabel": "B",
            "horsePower": 72,
            "acceleration0to100": 14.5
          }
        },
        {
          "variantName": "X-cite CVT",
          "engineSpecification": "1.0 VVT-i - 72 hk",
          "transmission": "cvt",
          "pricing": {
            "monthlyPayment": 3199,
            "firstPayment": 9597,
            "annualKilometers": 15000,
            "co2TaxBiannual": 330
          },
          "specifications": {
            "fuelConsumptionKmpl": 18.5,
            "co2EmissionsGkm": 123,
            "energyLabel": "B",
            "horsePower": 72,
            "acceleration0to100": 14.5
          }
        },
        {
          "variantName": "X-clusiv CVT",
          "engineSpecification": "1.0 VVT-i - 72 hk",
          "transmission": "cvt",
          "pricing": {
            "monthlyPayment": 3399,
            "firstPayment": 10197,
            "annualKilometers": 15000,
            "co2TaxBiannual": 330
          },
          "specifications": {
            "fuelConsumptionKmpl": 18.5,
            "co2EmissionsGkm": 123,
            "energyLabel": "B",
            "horsePower": 72,
            "acceleration0to100": 14.5
          }
        }
      ]
    }
  ],
  "accessories": [
    {
      "packageName": "Vinterhjul 15\"",
      "monthlyCost": 299,
      "category": "wheels"
    },
    {
      "packageName": "Vinterhjul 16\"",
      "monthlyCost": 349,
      "category": "wheels"
    },
    {
      "packageName": "Alufælge 17\" sportspakke",
      "monthlyCost": 450,
      "category": "wheels"
    },
    {
      "packageName": "Service & Vedligeholdelse",
      "monthlyCost": 450,
      "category": "service"
    },
    {
      "packageName": "Kasko forsikring",
      "monthlyCost": 650,
      "category": "insurance"
    },
    {
      "packageName": "Ansvarsforsikring",
      "monthlyCost": 150,
      "category": "insurance"
    },
    {
      "packageName": "Metallic lak",
      "monthlyCost": 125,
      "category": "other"
    },
    {
      "packageName": "Tågelygter",
      "monthlyCost": 75,
      "category": "other"
    },
    {
      "packageName": "Aircondition",
      "monthlyCost": 200,
      "category": "other"
    }
  ],
  "metadata": {
    "extractionTimestamp": "2024-06-24T12:00:00.000Z",
    "documentPages": 1,
    "extractionWarnings": []
  }
}